### PR TITLE
Fix coretest

### PR DESCRIPTION
**Pull request by Scienfitz**
- removes unintended `examples` dependency of coretest
- this means it's not testing `onnx` surrogate things, substance parameter stuff and streamlit

### CODE REVIEW

File: tests/serialization/test_parameter_serialization.py [hunk 1]
```diff
@@ -7,27 +7,30 @@
...
     task_parameter,
 )
 
+parameter_strategies = [
+    param(numerical_discrete_parameter(), id="NumericalDiscreteParameter"),
+    param(numerical_continuous_parameter(), id="NumericalContinuousParameter"),
+    param(categorical_parameter(), id="CategoricalParameter"),
+    param(task_parameter(), id="TaskParameter"),
+    param(custom_parameter(), id="CustomParameter"),
+]
```
Comment (Scienfitz) [inline tests/serialization/test_parameter_serialization.py:25]: @AdrianSosic a first try using your suggested way via skipif (which I would also prefer) didnt work. It seems during the collection the object is still instantiated, resulting in imprt error even if its marked as skip

**Review by AVHopp — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,12 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Top-level example summaries
 
 ### Changed
-- Optional `chem` dependencies are lazily imported
 - Order of README sections
 
 ### Fixed
 - Several minor issues in documentation
 - Visibility and constructor exposure of `Campaign` attributes that should be private
+- Optional `chem` dependencies are lazily imported, which improves startup time and
```
[the remark this answers is not in the document]
Comment (AdrianSosic) [reply CHANGELOG.md:19]: Tried to keep it shorter, but if you leave the part after away, it's unclear what has actually been fixed. It's really two independent things, and the last one is even a bug. So, suggestions?

[the remark this answers is not in the document]
Comment (AVHopp) [reply CHANGELOG.md:19]: I'd split it in changed and FIxed:
Changed:
- Improved startup time by lazily importing optional chem dependencies

Fixed:
- Fix baybe import from environments containing core dependencies only by lazily loading chem dependencies.

This adds some redundancy, but as you said yourself: These are "really two independent things", and I'd prefer to have these as clearly separated points. But if you prefer to just have one, feel free to just resolve :)

[the remark this answers is not in the document]
Comment (Scienfitz) [reply CHANGELOG.md]: the split should be fine and clear so i changed it and will proceed to merge

**Pull request merged**